examples/jsm/utils/BufferGeometryUtils.js

```diff
@@ -10,10 +10,15 @@ import {
 	TrianglesDrawMode,
 	Vector3,
 } from 'three';
-import { generateTangents } from '../libs/mikktspace.module.js';
```

Comment:
It technically could, I think it's a question of the preferred API / usage:

```js
import * as MikkTSpace from 'three/examples/jsm/libs/mikktspace.module.js';
import { computeTangents } from 'three/examples/jsm/utils/BufferGeometryUtils.js';

await MikkTSpace.ready;

// (A)
geometry = computeTangents( geometry, MikkTSpace );

// (B)
geometry = computeTangents( geometry );
```

I was thinking (A) might be better because it is more explicit – you have to know about this MikkTSpace module, because you're passing it into the function, so there's perhaps less chance of forgetting to check `.ready` and having a subtle race condition. This also allows users to provide the [`mikktspace`](https://www.npmjs.com/package/mikktspace) npm package instead, which is more optimized for modern bundlers (it doesn't embed the WASM as a base64 string as required here).

I'm open to either (A) or (B), though.

Reply:
Right, if it makes users able to use the wasm version, it makes sense.